content/docs/reference-events.md

```diff
@@ -244,17 +244,17 @@ string pointerType
 boolean isPrimary
 ```
 
-A note on cross-browser support:
+Catatan untuk dukungan lintas _browser_:
 
-Pointer events are not yet supported in every browser (at the time of writing this article, supported browsers include: Chrome, Firefox, Edge, and Internet Explorer). React deliberately does not polyfill support for other browsers because a standard-conform polyfill would significantly increase the bundle size of `react-dom`.
+_Event_ penunjuk belum didukung oleh semua browser (saat penulisan artikel ini, _browser_ yang mendukung terdiri atas: Chrome, Firefox, Edge, dan Internet Explorer). React tidak mengisi dukungan untuk _browser_ lain karena akan meningkatkan ukuran berkas `react-dom` secara singnifikan.
```

Comment:
```suggestion
_Pointer event_ belum didukung oleh semua _browser_ (saat penulisan artikel ini, _browser_ yang telah mendukung adalah: Chrome, Firefox, Edge, dan Internet Explorer). React tidak serta-merta memberikan dukungan untuk _browser_ lain karena akan meningkatkan ukuran bundel `react-dom` secara singnifikan.
```

Since _pointer events_ in the earlier section doesn't get translated I think it's better if we leave it as it is for all the subsequent occurence to maintain clarity.

What do you think? 😄

Reply:
:+1: . also fixed the typo "singnifikan" -> "signifikan"